.github/workflows/rust.yml

```
@@ -2,8 +2,10 @@ name: Rust
 
 on:
   push:
-    branches: [main]
+    branches: [main, "release-dry-run*"]
```

I think we should rename this to `release/**`, and plan to use it for a release branches in general, not just as dry runs.
```suggestion
    branches: [main, "release/**"]
```